### PR TITLE
Improve docs for gen-driftignore command

### PR DESCRIPTION
**Pull request by sundowndev**

### CODE REVIEW

File: docs/usage/cmd/gen-driftignore.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+id: gen-driftignore-usage
+title: Driftignore generator
+---
+
+Many of us have reasons to not target 100% IaC coverage of our AWS accounts. In this case, driftctl continuously delivers drift notifications that we don’t care about. Behold! There’s a solution.
```
Comment (wbeuil) [inline docs/usage/cmd/gen-driftignore.mdx:6]: `Some people do not have the goal of reaching a 100% IAC coverage with their infrastructure. And for them, driftctl can be annoying to continuously deliver drift notifications on resources they don't care. For this use case, there's a solution.`

**Review by lotoussa — APPROVED** [review]
lgtm

File: docs/usage/cmd/gen-driftignore.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+You can start using driftctl with a clean state, by ignoring all the current resources that are not yet under control. Driftctl provides a command to automatically generate a driftignore file from a previous scan. This command parses a JSON input from a given file and sends output to stdout.
```
Comment (wbeuil) [inline docs/usage/cmd/gen-driftignore.mdx:8]: Don't put driftctl with an uppercase letter and don't forget the . on .driftignore

`driftctl provides a command to automatically generate a .driftignore file from a previous scan given the type of resources you want to exclude (e.g. unmanaged, missing or changed resources). This command parses a JSON input from a given file and sends output to stdout.`

File: docs/usage/cmd/gen-driftignore.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+# Changed resources will be excluded
+$ driftctl scan --from tfstate://state1.tfstate -o json://stdout | driftctl gen-driftignore -i /dev/stdin --exclude-changed
+
+# Unmanaged resources will be excluded, output sent to .driftignore file
+$ driftctl scan --from tfstate://state1.tfstate -o json://result.json
+$ driftctl gen-driftignore -i result.json --exclude-unmanaged > .driftignore
+```
+
+You can filter which kind of resource you want to ignore using these flags :
```
Comment (wbeuil) [inline docs/usage/cmd/gen-driftignore.mdx:22]: `which kind of resources you ... flags:`

File: docs/usage/cmd/gen-driftignore.mdx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  -v ~/.aws:/root/.aws:ro \
+  -v $(pwd):/app \
+  -v ~/.driftctl:/root/.driftctl \
+  -e AWS_PROFILE=cloudskiff-demo \
+  -e AWS_REGION=us-east-1 \
+  cloudskiff/driftctl scan --from tfstate://**/*.tfstate --output json://drifts.json
+[...]
+```
+
+Finally generate the `.driftignore` file from the JSON:
```
Comment (wbeuil) [inline docs/usage/cmd/gen-driftignore.mdx:43]: `Finally, ...`

**Review by wbeuil — CHANGES_REQUESTED** [review]

**Review by wbeuil — APPROVED** [review]

**Pull request merged**